draft-ietf-rats-corim.md

```diff
@@ -1780,70 +1787,120 @@ Regardless of the specific integrity protection method used, the Evidence's inte
 
 ### Input Transformation {#sec-phase1-trans}
 
-Inputs, whether Endorsements, Reference Values, Evidence, or Policies, are transformed to an internal representation that is based on ECTs.
+Input Conceptual Messages, whether Endorsements, Reference Values, Evidence, or Policies, are transformed to an internal representation that is based on ECTs ({{sec-ir-cm}}).
 
 The following mapping conventions apply to all forms of input transformation:
-The `e` field is populated with a Target Environment identifier.
-The `c` field is populated with the measurements collected by an Attesting Environment.
-The `a` field is populated with the identity of the entity that asserted (e.g., signed) the Evidence.
-The `ns` field is populated with the namespace context if supplied. For example, the Attester's manufacturer may have a URI that identifies the manufacturing series, family or architecture.
-The `cm` field is set based on the type of Conceptual Message inputted or to be outputed.
+
+> * The `environment` field is populated with a Target Environment identifier.
+> * The `element-list` field is populated with the measurements collected by an Attesting Environment.
+> * The `authority` field is populated with the identity of the entity that asserted (e.g., signed) the Conceptual Message.
+> * The `cmtype` field is set based on the type of Conceptual Message inputted or to be outputed.
+> * The `profile` field is set based on the `corim-map` `profile` value.
 
 #### Appraisal Context Construction
 
 All of the extracted and validated tags are loaded into an *appraisal context*.
 The Appraisal Context contains an internal representation of the inputted Conceptual Messages.
 The selected tags are mapped to an internal representation, making them suitable for appraisal processing.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/96
-
 #### Reference Triples Transformation {#sec-ref-trans}
 
 {:rtt-enum: counter="foo" style="format Step %d."}
 
 {: rtt-enum}
-* An available `rv` list entry ({{sec-ir-ref-val}}) is allocated.
-The Reference Values Triple ({{sec-comid-triple-refval}}) `environment-map` is copied to the `environment-map` for both the `rv` `condition` and `rv` `addition` ECTs.
+* An `rv` list entry ({{sec-ir-ref-val}}) is allocated.
```

Comment:
If we allow reference value triples to apply to endorsed measurements, then the `rv` relation is the same as the `ev` relation. I would say we don't need it.

Reply:
CoRIM is written from the perspective that is laid out by RFC9334 where the roles of Reference Value Provider and Endorser differ. The interaction between RVPs, Attesters and RPs is meaningful because of the conceptual messages they exchange. There isn't the notion of a role credential. The result of an RVP-Attester interaction is "corroborated" Ref Values that have cmtype=`reference` but is an "actual state" representation in the ACS. If an Endorser constructed an endorsement rule that produced a record with the same claims as a corroborated RV contains, it would still be an Endorsement. RFC9334 expects RPs will differentiate between the roles and will be looking for statements from RVPs regarding acceptability of Evidence vs. a similar construction from Endorsers.

Reply:
I understand that the roles are different, but the behavior of `rv` as an internal representation can easily be expressed as `ev` since you're already populating the additions with the appropriate `cmtype`. That's all I was saying.

Reply:
> I understand that the roles are different, but the behavior of `rv` as an internal representation can easily be expressed as `ev` since you're already populating the additions with the appropriate `cmtype`. That's all I was saying.

The internal representation is primarily there to facilitate readability. Having distinct representations for the various conceptual message types keeps the text inside the conceptual message swim lanes. It's a style thing more than an optimization thing.